### DISCUSSION TITLE
docs/v1.11.x/en/ops/kubectl-ko/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#3513] [Announcements]
# docs/v1.11.x/en/ops/kubectl-ko/

https://kubeovn.github.io/docs/v1.11.x/en/ops/kubectl-ko/

**Comment by WindHenry** [comment]
You know we can get k8s info resource like endpoint:port/api/v1/..../pods. Could we get ovn info by that way? Thank you!

**Reply by oilbeater** [reply]
kubectl-ko use exec to retrieve ovn information underground, you can find the detail implementation in https://github.com/kubeovn/kube-ovn/blob/master/dist/images/kubectl-ko.

**Reply by WindHenry** [reply]
Oh Thank you! You saved me!!!
